tests/test_schema.py

```diff
@@ -106,6 +121,35 @@ def test_dump_to(self, post):
         assert 'relationships' in data['data']
         assert 'post-comments' in data['data']['relationships']
 
+
+class TestCompoundDocuments:
+
+    def test_include_data_with_many(self, post):
+        data = PostSchema(include_data=('post_comments',)).dump(post).data
+        assert 'included' in data
+        assert len(data['included']) == 2
+        first_comment = data['included'][0]
+        assert 'attributes' in first_comment
+        assert 'body' in first_comment['attributes']
+
+    def test_include_data_with_single(self, post):
+        data = PostSchema(include_data=('author',)).dump(post).data
+        assert 'included' in data
+        assert len(data['included']) == 1
+        author = data['included'][0]
+        assert 'attributes' in author
+        assert 'first_name' in author['attributes']
+
+    def test_include_data_with_all_relations(self, post):
+        data = PostSchema(include_data=('author', 'post_comments')).dump(post).data
+        assert 'included' in data
+        assert len(data['included']) == 3
+
+    def test_include_no_data(self, post):
+        data = PostSchema(include_data=()).dump(post).data
+        assert 'included' not in data
+
+
```

Comment:
Could you give me an example of a scenario where a validation fails? I'm having a hard time trying to provoke a ValidationError when dumping (and this compound documents are dumped only).